tests/wpt/css-tests/cssom-1_dev/html/index-002.htm

```diff
@@ -30,7 +30,7 @@
       ],
 
       'overflow': [
-        ['overflow-x: scroll; overflow-y: hidden;', 'overflow: scroll hidden;'],
+        ['overflow-x: scroll; overflow-y: hidden;', 'overflow-x: scroll; overflow-y: hidden;'],
```

Comment:
Good to know, thanks. So I should undo my change here, and simply set the above to expected FAIL, right?